lib/statusbar-summary-view.coffee

```diff
@@ -27,16 +27,24 @@ class StatusBarSummaryView
     @tile = statusBar.addRightTile({item: el, priority: 100})
 
 class StatusBarSummary extends View
+  initialize: ->
+    @on 'click', '.linter-summary-container', ->
+      console.log("moveToNextMessage")
```

Comment:
Hi @dmnd , I would like to use the `moveToNextMessage` function that you made. (https://github.com/AtomLinter/Linter/blob/master/lib/linter-view.coffee#L233)

Should I move it to libs/utils.js ? Or do you prefer if I make my own function?

I would like to avoid duplicate code.